middlewares/stripPrefix.go

```diff
@@ -1,6 +1,7 @@
 package middlewares
 
 import (
+	"github.com/containous/traefik/log"
```

Comment:
Can you add the non-stdlib imports after the stdlib ones, separated by a blank line?